myautoload.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
my custom autoload file

Reply:
Neither of those is needed, should be removed

Reply:
not sure what you mean exactly. 

I need a custom autoloader in my real world app, therefore I tried to mimic it here with this very simple script so I can reproduce the actual issue